src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -137,24 +142,28 @@ import { useStore } from '@/store/store'
 import { CommonMixin } from '@/mixins'
 import { BusinessLookupServices, LegalServices } from '@/services'
 import { BusinessLookup } from '@bcrs-shared-components/business-lookup'
-import { BusinessIF, BusinessLookupIF, EmptyBusinessLookup } from '@/interfaces'
+import { BusinessLookupIF, EmptyBusinessLookup } from '@/interfaces'
+import BusinessTable from '@/components/Amalgamation/BusinessTable.vue'
 
 @Component({
   components: {
-    BusinessLookup
+    BusinessLookup,
+    BusinessTable
   }
 })
 export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
-  @Getter(useStore) getAmalgamatingBusinesses!: Array<BusinessIF>
+  @Getter(useStore) getAmalgamatingBusinesses!: Array<any>
+  @Getter(useStore) getShowErrors!: boolean
   @Getter(useStore) isAmalgamationFilingHorizontal!: boolean
   @Getter(useStore) isRoleStaff!: boolean
 
-  @Action(useStore) setAmalgamatingBusinesses!: (x: Array<BusinessIF>) => void
+  @Action(useStore) setAmalgamatingBusinesses!: (x: Array<any>) => void
 
   // Local properties
   amalgamatingBusinessesValid = false
   amalgamatingBusinesses = []
   initialBusinessLookupObject = EmptyBusinessLookup
+  businessTableValid = false
```

Comment:
Note that initial value here doesn't matter since the BusinessTable component computes this early on and emits its validity.